projects/packages/jetpack-mu-wpcom/src/features/wpcom-block-editor-nux/src/disable-core-welcome-guide.js

```diff
@@ -16,12 +13,7 @@ const unsubscribe = subscribe( () => {
 } );
```

Comment:
I'm curious about this file and why there are two very similar subscribe calls. 🤔 

Seems like a good idea not to change any business logic in this PR.